client/post-editor/media-modal/gallery/fields.jsx

```diff
@@ -15,15 +16,17 @@ import EditorMediaModalFieldset from '../fieldset';
 import SelectDropdown from 'components/select-dropdown';
 import SelectDropdownItem from 'components/select-dropdown/item';
 import FormCheckbox from 'components/forms/form-checkbox';
+import Button from 'components/button';
 import { GalleryColumnedTypes, GallerySizeableTypes } from 'lib/media/constants';
 
-export default React.createClass( {
+export const EditorMediaModalGalleryFields = React.createClass( {
 	displayName: 'EditorMediaModalGalleryFields',
```

Comment:
When assigning `React.createClass` to a variable, you can remove `displayName`, as it's inferred automatically via [`babel-plugin-react-display-name`](https://www.npmjs.com/package/babel-plugin-react-display-name).